integration/jest.sequencer.js

```diff
@@ -0,0 +1,10 @@
+const Sequencer = require("@jest/test-sequencer").default;
+
+class CustomSequencer extends Sequencer {
+  sort(tests) {
+    const copyTests = Array.from(tests);
+    return copyTests.sort((testA, testB) => (testA.path > testB.path ? 1 : -1));
```

Comment:
Despite the tests use to be executed in alphabetical order, there is no actual guarantee of that. It was causing a failure in the multicluster test because the secret created in the secret-repo test wasn't being found.
This sequencer just ensures this alphabetical order (I've pushed the integration img with this dep as well)

Reply:
We don't yet support app repos (let alone secret ones) on other clusters (#1982)  because the interface to sync with the asset svc is via the DB (#2394) (left over from the read-only monocular).

Reply:
Thanks for the clarification, I always forget about this issue. 
Perhaps we should name the tests like "01-xxxx.js", "00-xxx.js" just to make clear that they are being executed in order.

Reply:
I know that private repositories are not supported in the second cluster but what I don't understand is why the order of the tests affects the result. I thought that creating a private repo in the default cluster caused an error when trying to deploy in the second cluster but that may not be it.

Reply:
Ok, ok, I see your point now and it may be an issue as you said:

When creating a private repo first, we are creating the docker credentials. Somehow, when deploying in the second cluster, it tries to retrieve these credentials (not available, though), and fails, therefore.

![image](https://user-images.githubusercontent.com/11535726/113563658-60086600-9608-11eb-9c82-ebe7b68d05bf.png)

Let me replicate the issue in my dev environment and will file an issue with the details.

However, with regards to this PR, I think (for the sake of replicability) to still define an explicit order during the test suite execution.

Reply:
I wasn't able to reproduce this issue in my dev environment. I set up the multicluster env and: 1) tried to manually create the private repo and then a new deployment; 2) run the e2e test directly from `yarn start ...`.
In both cases, the credentials are being properly stored and the app deploys seamlessly as well.

That's odd... :S

Reply:
You probably added a namespaced private repository. If you create it in the `kubeapps` namespace, you are likely able to reproduce the issue.